README.md

```diff
@@ -152,6 +152,28 @@ https://www.morphosource.org/downloads/000425163?file=thumbnail&t=1645803720
 
 If the media id isn't found a `morphosource.api.ItemNotFound` exception will be raised.
 
+#### Get Media Metadata
+The `Media` object method `get_file_metadata()` can be used to retrieve file metadata for the media object.
+
+In this example we fetch media with id "000425163" and lookup file metadata information. Note that the returned file size is given in bytes.
+The `data` property contains all fields returned from the MorphoSource API.
+```python
+from morphosource import get_media
+
+media = get_media(media_id="000425163")
+metadata = media.get_file_metadata()
+print("Filename:", metadata.file_name)
+print("File size:", metadata.file_size)
```

Comment:
```suggestion
print("File size:", metadata.file_size)
print("Contents mime type:", metadata.contents_mime_type)
```

To match the example output. Not sure if we want to do `mime_type`, seems unnecessary, but could.

Reply:
Oops I mean to remove that output.